### ISSUE TITLE
Feature request - Animus 3 - Ability to change led colors at runtime

### ISSUE THREAD

**Issue by BlackCapCoder** [blahlicus/animus-family#28]
LEDs are currently hardcoded to a neat rainbow color:

https://github.com/blahlicus/animus-family/blob/81f074a83722be4aeb501e9906576d4714493cb1/animus-3/source/animus/ModLED.cpp#L37-L44

It would be nice to be able to change this at runtime, rather than just the brightness. Ideally I would like to change the color in response to a change in active layer.

**Comment by blahlicus** [comment]
The capability to do so is already in animus, the desktop software configurator tool arbites is the one lacking the feature to configure it in animus. It is not hard coded, `GetData` reads data from the same persistent memory that stores the key layout.

RefreshLED is called every time the LoadData event handler is called. LoadData is called when persistent memory is changed (this happens when you upload a new key layout to your keyboard via animus)

RefreshLED reads a HSV value from persistent memory to display the appropriate hue and saturation when it is called so the capability to change the colour is already in the firmware. Arbites configures the persistent memory every time a layout is uploaded, so Arbites could configure the RGB LEDs if the appropriate communication messages were set up.

I'll post a rudimentary software to configure your RGB LEDs later today once I've finished debugging it, I'll comment here again once I've posted it.
